src/QMCWaveFunctions/tests/test_TrialWaveFunction.cpp

```diff
@@ -74,21 +77,23 @@ TEST_CASE("TrialWaveFunction", "[wavefunction]")
   elec_.addTable(ions_, DT_AOS);
 #endif
   elec_.resetGroups();
-  elec_.update();
   elec_.createSK(); // needed by AoS J2 for ChiesaKEcorrection
 
   ParticleSetPool ptcl{c};
   ptcl.addParticleSet(&elec_);
   ptcl.addParticleSet(&ions_);
 
+  // make a ParticleSet Clone
+  ParticleSet elec_clone(elec_);
+
   //diamondC_1x1x1
-  const char* particles = "<tmp> \
-<determinantset type=\"einspline\" href=\"pwscf.pwscf.h5\" tilematrix=\"1 0 0 0 1 0 0 0 1\" twistnum=\"0\" source=\"ion\" meshfactor=\"1.0\" precision=\"float\" size=\"4\"/> \
+  const char* spo_xml = "<tmp> \
```

Comment:
https://en.cppreference.com/w/cpp/language/string_literal
I think the C++11 raw string literal looks better in the test source.  XML can be multiline and doesn't need tons of escaping and as such its easier to see what the test does.

Reply:
I have limited knowledge on this. I think we need a dedicated PR to handle all the hits of this pattern in the tests.